app/helpers/requests/application_helper.rb

```diff
@@ -155,7 +164,11 @@ def pickup_choices requestable, default_pickups
         end
         # id = requestable.item? ? requestable.item['id'] : requestable.holding['id']
         content_tag(:div, id: "fields-print__#{requestable.preferred_request_id}", class: class_list) do
-          locs = self.available_pickups(requestable, default_pickups)
+          if requestable.services.include? 'in_process'
```

Comment:
Yes, that was an oversight on my part. I meant to add the `on_order` use case in. Most recent commit fixes it, and also covers the case when the `holding_library` is `nil`.